src/ia/js/IAPage.js

```diff
@@ -242,14 +261,26 @@
             'other_queries'
         ],
 
+        updateData: function(ia_data, x, edited) {
+            $.each(ia_data.live, function(key, value) {
+                if (edited) {
+                    x[key] = ia_data.edited[key]? ia_data.edited[key] : value;
+                } else {
+                    x[key] = value;
```

Comment:
@jagtalon Thing is, directly assigning ia_data.live to x clones the object, it doesn't just copy it. So if the user later switches again to edited and x changes again, ia_data.live will be changed as well.